doc/languages-frameworks/coq.section.md

```diff
@@ -3,37 +3,37 @@
 ## Coq derivation: `coq`
 
 The Coq derivation is overridable through the `coq.override overrides`, where overrides is an attribute set which contains the arguments to override. We recommend overriding either of the following
-+ `version` (optional, defaults to the latest version of Coq selected for nixpkgs, see `pkgs/top-level/coq-packages` to witness this choice), which follows the conventions explained in the `coqPackages` section below,
```

Comment:
You would likely need an empty line between the paragraph and the list item. This is why I prefer Docbook over Markdown :laughing: